packages/drivers/odsp-driver/src/createFile.ts

```diff
@@ -156,6 +160,22 @@ export async function createNewFluidFileFromSummary(
             },
             { end: true, cancel: "error" });
     });
+
+    const odspUrl = createOdspUrl({... newFileInfo, itemId, dataStorePath: "/"});
+    const resolver = new OdspDriverUrlResolver();
+
+    // converting summary and getting sequence number
+    const snapshot: IOdspSnapshot = convertSummaryTreeToIOdspSnapshot(createNewSummary);
```

Comment:
This is more feedback to @jatgarg, I think we need to move forward as is, but I think we need to revisit this conversion as a next step.
Given that we are building new wire format for snapshots, we should reduce our dependency on IOdspSnapshot, and thus not add (but rather remove) conversions like convertSummaryTreeToIOdspSnapshot. Instead we should change caching format to be closer to what runtime needs (ISnapshotTree + blobs in some form) and not force more code paths to depend on IOdspSnapshot. I'd recommend to refactor existing code to reduce usage of IOdspSnapshot to a minimum, basically isolate existing file format path to a leaf code path - something that can be removed once we have new wire format.

Reply:
Yes, I will look into that.